docker/README.md

```diff
@@ -3,23 +3,49 @@
 
 You can build a docker image based on this Dockerfile. This image will contain only an Orion Context Broker instance, exposing port `1026`. This requires that you have [docker](https://docs.docker.com/installation/) installed on your machine.
 
-If you just want to have an Orion Context Broker running as fast as possible jump to section [The Fastest Way](#the_fastest_way).
+If you just want to have an Orion Context Broker running as fast as possible jump to section *The Fastest Way*.
 
-If you want to know what and how we are doing things with the container step by step you can  go ahead and read the [Build](#build_the_image) and [Run](#run_the_container) sections.
+If you want to know what and how we are doing things with the container step by step you can go ahead and read the build and run sections.
 
 ## The Fastest Way
 
-A Docker Compose file is provided for convenience. You must install [Docker Compose](https://docs.docker.com/compose/install/) for this method to work.
+Docker Compose lets you have an Orion Context Broker container along with a MongoDB container running in a few minutes. You must install [Docker Compose](https://docs.docker.com/compose/install/) for this method to work.
 
-Simply navigate to the docker directory of the fiware-orion code (if you have downloaded it) and run
+### Run a container from an image you just built
+
+If you have downloaded the [Orion's code](https://github.com/telefonicaid/fiware-orion/) simply navigate to the docker directory and run
+
+	docker-compose up
+
+You are using this [docker-compose.yml](https://github.com/telefonicaid/fiware-orion/blob/develop/docker/docker-compose.yml) file. This will instruct to build a new image for Orion and run it linking it to another MongoDB container.
+
+### Run a container pulling an image from the cloud (recommended)
+
+If you haven't or you don't want to download the whole thing, you can create a file called `docker-compose.yaml` in a directory of your choice and fill it with the following content
+
+	orion:
```

Comment:
Any not manual way of ensuring that this fragment is exactly the same than the content of the docker-compose.yml? Maybe using check_file_compliance.py?